mm_image_processing/util/image_processor.py

```diff
@@ -5,15 +5,16 @@
     - processImgFromLocal : process image from local directory
     - buildGeoJson : convert classified data to geojson
 """
-from rasterio.plot import reshape_as_image, reshape_as_raster
+import geopandas as gpd
+import numpy as np
+import logging
 import rasterio
 import rioxarray
-from util import features
+
+from rasterio.plot import reshape_as_image, reshape_as_raster
 from rasterio.features import shapes
-import geopandas as gpd
-import numpy as np
+from mm_image_processing.util import features 
 
-import logging
```

Comment:
Note that PEP8 suggests the following ordering:

```
import logging

import geopandas as gpd
import numpy as np
import rasterio
import rioxarray
from rasterio.features import shapes
from rasterio.plot import reshape_as_image, reshape_as_raster

from mm_image_processing.util import features
```

Reply:
updated to PEP8 as well